clients/src/main/java/org/apache/kafka/clients/consumer/internals/metrics/KafkaConsumerMetrics.java

```diff
@@ -26,20 +26,20 @@
 
 import java.util.concurrent.TimeUnit;
 
-public class KafkaConsumerMetrics implements AutoCloseable {
+import static org.apache.kafka.clients.consumer.internals.metrics.AbstractConsumerMetricsManager.MetricGroupSuffix.CONSUMER;
+
+public class KafkaConsumerMetrics extends AbstractConsumerMetricsManager implements AutoCloseable {
     private final MetricName lastPollMetricName;
     private final Sensor timeBetweenPollSensor;
     private final Sensor pollIdleSensor;
     private final Sensor committedSensor;
     private final Sensor commitSyncSensor;
-    private final Metrics metrics;
     private long lastPollMs;
     private long pollStartMs;
     private long timeSinceLastPollMs;
 
     public KafkaConsumerMetrics(Metrics metrics, String metricGrpPrefix) {
```

Comment:
i'm not even sure why we have metricGrpPrefix from the beginning - it is not specified by the user and it seems to always be the default.